/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:283: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418: FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
  @autocast(enabled=False)
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin torch._C._to_dlpack. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0419 14:28:09.027711 47319186158848 torch/_dynamo/guards.py:2611] [4/1] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0419 14:28:09.027711 47319186158848 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:09.027711 47319186158848 torch/_dynamo/guards.py:2611] [4/1] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:28:09.046560 47319186158848 torch/_dynamo/guards.py:2611] [5/1] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0419 14:28:09.046560 47319186158848 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:09.046560 47319186158848 torch/_dynamo/guards.py:2611] [5/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:28:09.057941 47319186158848 torch/_dynamo/guards.py:2611] [6/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0419 14:28:09.057941 47319186158848 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:09.057941 47319186158848 torch/_dynamo/guards.py:2611] [6/1] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:28:09.093285 47319186158848 torch/_dynamo/guards.py:2611] [8/1] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0419 14:28:09.093285 47319186158848 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:09.093285 47319186158848 torch/_dynamo/guards.py:2611] [8/1] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
W0419 14:28:15.815507 47319186158848 torch/_dynamo/variables/tensor.py:715] [30/0] Graph break from `Tensor.item()`, consider setting:
W0419 14:28:15.815507 47319186158848 torch/_dynamo/variables/tensor.py:715] [30/0]     torch._dynamo.config.capture_scalar_outputs = True
W0419 14:28:15.815507 47319186158848 torch/_dynamo/variables/tensor.py:715] [30/0] or:
W0419 14:28:15.815507 47319186158848 torch/_dynamo/variables/tensor.py:715] [30/0]     env TORCHDYNAMO_CAPTURE_SCALAR_OUTPUTS=1
W0419 14:28:15.815507 47319186158848 torch/_dynamo/variables/tensor.py:715] [30/0] to include these operations in the captured graph.
W0419 14:28:15.815507 47319186158848 torch/_dynamo/variables/tensor.py:715] [30/0] 
V0419 14:28:18.983766 47319186158848 torch/_dynamo/guards.py:2611] [4/2] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0419 14:28:18.983766 47319186158848 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:18.983766 47319186158848 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 1566
V0419 14:28:18.983766 47319186158848 torch/_dynamo/guards.py:2611] [4/2] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:28:19.012659 47319186158848 torch/_dynamo/guards.py:2611] [5/2] [__recompiles] Recompiling function to_dgl_nd in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/__init__.py:141
V0419 14:28:19.012659 47319186158848 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:19.012659 47319186158848 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 1566
V0419 14:28:19.012659 47319186158848 torch/_dynamo/guards.py:2611] [5/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:28:19.024371 47319186158848 torch/_dynamo/guards.py:2611] [6/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0419 14:28:19.024371 47319186158848 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:19.024371 47319186158848 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 0, actual 1566
V0419 14:28:19.024371 47319186158848 torch/_dynamo/guards.py:2611] [6/2] [__recompiles]     - tensor 'L['data']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:28:19.063657 47319186158848 torch/_dynamo/guards.py:2611] [8/2] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0419 14:28:19.063657 47319186158848 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:19.063657 47319186158848 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 1566
V0419 14:28:19.063657 47319186158848 torch/_dynamo/guards.py:2611] [8/2] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:28:19.081171 47319186158848 torch/_dynamo/guards.py:2611] [32/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0419 14:28:19.081171 47319186158848 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:19.081171 47319186158848 torch/_dynamo/guards.py:2611] [32/1] [__recompiles]     - L['___stack0'] == 768                                       
V0419 14:28:19.303608 47319186158848 torch/_dynamo/guards.py:2611] [34/1] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0419 14:28:19.303608 47319186158848 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:19.303608 47319186158848 torch/_dynamo/guards.py:2611] [34/1] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 1566
V0419 14:28:20.039788 47319186158848 torch/_dynamo/guards.py:2611] [20/1] [__recompiles] Recompiling function add_edges in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:435
V0419 14:28:20.039788 47319186158848 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:20.039788 47319186158848 torch/_dynamo/guards.py:2611] [20/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 1566, actual 427
V0419 14:28:20.061960 47319186158848 torch/_dynamo/guards.py:2611] [26/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_541 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:541
V0419 14:28:20.061960 47319186158848 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:20.061960 47319186158848 torch/_dynamo/guards.py:2611] [26/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 1566, actual 427
V0419 14:28:20.089102 47319186158848 torch/_dynamo/guards.py:2611] [27/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_542 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:542
V0419 14:28:20.089102 47319186158848 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:20.089102 47319186158848 torch/_dynamo/guards.py:2611] [27/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 1566, actual 427
V0419 14:28:20.105250 47319186158848 torch/_dynamo/guards.py:2611] [28/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_545 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:545
V0419 14:28:20.105250 47319186158848 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:20.105250 47319186158848 torch/_dynamo/guards.py:2611] [28/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 1566, actual 427
V0419 14:28:20.170717 47319186158848 torch/_dynamo/guards.py:2611] [30/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0419 14:28:20.170717 47319186158848 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:20.170717 47319186158848 torch/_dynamo/guards.py:2611] [30/1] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 1566, actual 427
V0419 14:28:20.421904 47319186158848 torch/_dynamo/guards.py:2611] [32/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_574 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:574
V0419 14:28:20.421904 47319186158848 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:20.421904 47319186158848 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 1566, actual 427
V0419 14:28:20.421904 47319186158848 torch/_dynamo/guards.py:2611] [32/2] [__recompiles]     - L['___stack0'] == 768                                       
V0419 14:28:20.665666 47319186158848 torch/_dynamo/guards.py:2611] [34/2] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0419 14:28:20.665666 47319186158848 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:20.665666 47319186158848 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 1566, actual 427
V0419 14:28:20.665666 47319186158848 torch/_dynamo/guards.py:2611] [34/2] [__recompiles]     - tensor 'L['seq'][0]' size mismatch at index 0. expected 0, actual 3132
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin posix.stat. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0419 14:28:24.957435 47319186158848 torch/_dynamo/guards.py:2611] [6/3] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0419 14:28:24.957435 47319186158848 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:24.957435 47319186158848 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0419 14:28:24.957435 47319186158848 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0419 14:28:24.957435 47319186158848 torch/_dynamo/guards.py:2611] [6/3] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0419 14:28:25.003156 47319186158848 torch/_dynamo/guards.py:2611] [7/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_at_435 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:435
V0419 14:28:25.003156 47319186158848 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:25.003156 47319186158848 torch/_dynamo/guards.py:2611] [7/1] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/torch/_dynamo/variables/functions.py:663: UserWarning: Graph break due to unsupported builtin time.perf_counter. This function is either a Python builtin (e.g. _warnings.warn) or a third-party C/C++ Python extension (perhaps created with pybind). If it is a Python builtin, please file an issue on GitHub so the PyTorch team can add support for it and see the next case for a workaround. If it is a third-party C/C++ Python extension, please either wrap it into a PyTorch-understood custom operator (see https://pytorch.org/tutorials/advanced/custom_ops_landing_page.html for more details) or, if it is traceable, use torch.compiler.allow_in_graph.
  torch._dynamo.utils.warn_once(msg)
V0419 14:28:25.242520 47319186158848 torch/_dynamo/guards.py:2611] [86/1] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:28:25.242520 47319186158848 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:25.242520 47319186158848 torch/_dynamo/guards.py:2611] [86/1] [__recompiles]     - L['___stack0'] == 200409.316400762                          
V0419 14:28:25.336323 47319186158848 torch/_dynamo/guards.py:2611] [86/2] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:28:25.336323 47319186158848 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:25.336323 47319186158848 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 200409.32917579                           
V0419 14:28:25.336323 47319186158848 torch/_dynamo/guards.py:2611] [86/2] [__recompiles]     - L['___stack0'] == 200409.316400762                          
V0419 14:28:28.537486 47319186158848 torch/_dynamo/guards.py:2611] [86/3] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:28:28.537486 47319186158848 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:28.537486 47319186158848 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 200409.422952508                          
V0419 14:28:28.537486 47319186158848 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 200409.32917579                           
V0419 14:28:28.537486 47319186158848 torch/_dynamo/guards.py:2611] [86/3] [__recompiles]     - L['___stack0'] == 200409.316400762                          
V0419 14:28:30.314556 47319186158848 torch/_dynamo/guards.py:2611] [86/4] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:28:30.314556 47319186158848 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.314556 47319186158848 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 200412.62403225                           
V0419 14:28:30.314556 47319186158848 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 200409.422952508                          
V0419 14:28:30.314556 47319186158848 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 200409.32917579                           
V0419 14:28:30.314556 47319186158848 torch/_dynamo/guards.py:2611] [86/4] [__recompiles]     - L['___stack0'] == 200409.316400762                          
V0419 14:28:30.384613 47319186158848 torch/_dynamo/guards.py:2611] [86/5] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:28:30.384613 47319186158848 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.384613 47319186158848 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 200414.400925719                          
V0419 14:28:30.384613 47319186158848 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 200412.62403225                           
V0419 14:28:30.384613 47319186158848 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 200409.422952508                          
V0419 14:28:30.384613 47319186158848 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 200409.32917579                           
V0419 14:28:30.384613 47319186158848 torch/_dynamo/guards.py:2611] [86/5] [__recompiles]     - L['___stack0'] == 200409.316400762                          
V0419 14:28:30.393252 47319186158848 torch/_dynamo/guards.py:2611] [86/6] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:28:30.393252 47319186158848 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.393252 47319186158848 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 200414.47121257                           
V0419 14:28:30.393252 47319186158848 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 200414.400925719                          
V0419 14:28:30.393252 47319186158848 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 200412.62403225                           
V0419 14:28:30.393252 47319186158848 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 200409.422952508                          
V0419 14:28:30.393252 47319186158848 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 200409.32917579                           
V0419 14:28:30.393252 47319186158848 torch/_dynamo/guards.py:2611] [86/6] [__recompiles]     - L['___stack0'] == 200409.316400762                          
V0419 14:28:30.458975 47319186158848 torch/_dynamo/guards.py:2611] [57/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0419 14:28:30.458975 47319186158848 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.458975 47319186158848 torch/_dynamo/guards.py:2611] [57/1] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:30.482029 47319186158848 torch/_dynamo/guards.py:2611] [61/1] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0419 14:28:30.482029 47319186158848 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.482029 47319186158848 torch/_dynamo/guards.py:2611] [61/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:30.501726 47319186158848 torch/_dynamo/guards.py:2611] [62/1] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0419 14:28:30.501726 47319186158848 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.501726 47319186158848 torch/_dynamo/guards.py:2611] [62/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:30.521511 47319186158848 torch/_dynamo/guards.py:2611] [63/1] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0419 14:28:30.521511 47319186158848 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.521511 47319186158848 torch/_dynamo/guards.py:2611] [63/1] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:30.539977 47319186158848 torch/_dynamo/guards.py:2611] [64/1] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0419 14:28:30.539977 47319186158848 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.539977 47319186158848 torch/_dynamo/guards.py:2611] [64/1] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:30.559606 47319186158848 torch/_dynamo/guards.py:2611] [65/1] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0419 14:28:30.559606 47319186158848 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.559606 47319186158848 torch/_dynamo/guards.py:2611] [65/1] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:30.606649 47319186158848 torch/_dynamo/guards.py:2611] [68/1] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0419 14:28:30.606649 47319186158848 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.606649 47319186158848 torch/_dynamo/guards.py:2611] [68/1] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:30.620338 47319186158848 torch/_dynamo/guards.py:2611] [70/1] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0419 14:28:30.620338 47319186158848 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.620338 47319186158848 torch/_dynamo/guards.py:2611] [70/1] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:30.638185 47319186158848 torch/_dynamo/guards.py:2611] [74/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0419 14:28:30.638185 47319186158848 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.638185 47319186158848 torch/_dynamo/guards.py:2611] [74/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:30.654349 47319186158848 torch/_dynamo/guards.py:2611] [77/1] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0419 14:28:30.654349 47319186158848 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.654349 47319186158848 torch/_dynamo/guards.py:2611] [77/1] [__recompiles]     - L['shape'][1] == 3703                                       
V0419 14:28:30.913811 47319186158848 torch/_dynamo/guards.py:2611] [6/4] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:432
V0419 14:28:30.913811 47319186158848 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.913811 47319186158848 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0419 14:28:30.913811 47319186158848 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:30.913811 47319186158848 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0419 14:28:30.913811 47319186158848 torch/_dynamo/guards.py:2611] [6/4] [__recompiles]     - GLOBAL_STATE changed: grad_mode 
V0419 14:28:30.969430 47319186158848 torch/_dynamo/guards.py:2611] [79/1] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0419 14:28:30.969430 47319186158848 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:30.969430 47319186158848 torch/_dynamo/guards.py:2611] [79/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 3703, actual 128
V0419 14:28:31.006690 47319186158848 torch/_dynamo/guards.py:2611] [80/1] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0419 14:28:31.006690 47319186158848 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:31.006690 47319186158848 torch/_dynamo/guards.py:2611] [80/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 3703, actual 128
V0419 14:28:31.016945 47319186158848 torch/_dynamo/guards.py:2611] [81/1] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0419 14:28:31.016945 47319186158848 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:31.016945 47319186158848 torch/_dynamo/guards.py:2611] [81/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 3703, actual 128
V0419 14:28:31.028879 47319186158848 torch/_dynamo/guards.py:2611] [82/1] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0419 14:28:31.028879 47319186158848 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:31.028879 47319186158848 torch/_dynamo/guards.py:2611] [82/1] [__recompiles]     - tensor 'L['input']' stride mismatch at index 0. expected 3703, actual 128
V0419 14:28:31.043685 47319186158848 torch/_dynamo/guards.py:2611] [83/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_165 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:165
V0419 14:28:31.043685 47319186158848 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:31.043685 47319186158848 torch/_dynamo/guards.py:2611] [83/1] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:31.427853 47319186158848 torch/_dynamo/guards.py:2611] [86/7] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:28:31.427853 47319186158848 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:31.427853 47319186158848 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200414.47978173                           
V0419 14:28:31.427853 47319186158848 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200414.47121257                           
V0419 14:28:31.427853 47319186158848 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200414.400925719                          
V0419 14:28:31.427853 47319186158848 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200412.62403225                           
V0419 14:28:31.427853 47319186158848 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200409.422952508                          
V0419 14:28:31.427853 47319186158848 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200409.32917579                           
V0419 14:28:31.427853 47319186158848 torch/_dynamo/guards.py:2611] [86/7] [__recompiles]     - L['___stack0'] == 200409.316400762                          
V0419 14:28:31.435481 47319186158848 torch/_dynamo/guards.py:2611] [86/8] [__recompiles] Recompiling function torch_dynamo_resume_in_hook_at_10 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10
V0419 14:28:31.435481 47319186158848 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:31.435481 47319186158848 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200415.514361805                          
V0419 14:28:31.435481 47319186158848 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200414.47978173                           
V0419 14:28:31.435481 47319186158848 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200414.47121257                           
V0419 14:28:31.435481 47319186158848 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200414.400925719                          
V0419 14:28:31.435481 47319186158848 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200412.62403225                           
V0419 14:28:31.435481 47319186158848 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200409.422952508                          
V0419 14:28:31.435481 47319186158848 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200409.32917579                           
V0419 14:28:31.435481 47319186158848 torch/_dynamo/guards.py:2611] [86/8] [__recompiles]     - L['___stack0'] == 200409.316400762                          
W0419 14:28:31.435621 47319186158848 torch/_dynamo/convert_frame.py:762] [86/8] torch._dynamo hit config.cache_size_limit (8)
W0419 14:28:31.435621 47319186158848 torch/_dynamo/convert_frame.py:762] [86/8]    function: 'torch_dynamo_resume_in_hook_at_10' (/work2/09808/mp46753/frontera/ML_Project/VQGraph/benchmark_utils.py:10)
W0419 14:28:31.435621 47319186158848 torch/_dynamo/convert_frame.py:762] [86/8]    last reason: L['___stack0'] == 200409.316400762                          
W0419 14:28:31.435621 47319186158848 torch/_dynamo/convert_frame.py:762] [86/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0419 14:28:31.435621 47319186158848 torch/_dynamo/convert_frame.py:762] [86/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:28:32.408783 47319186158848 torch/_dynamo/guards.py:2611] [0/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:449
V0419 14:28:32.408783 47319186158848 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:32.408783 47319186158848 torch/_dynamo/guards.py:2611] [0/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:32.438423 47319186158848 torch/_dynamo/guards.py:2611] [1/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:154
V0419 14:28:32.438423 47319186158848 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:32.438423 47319186158848 torch/_dynamo/guards.py:2611] [1/1] [__recompiles]     - tensor 'L['feats']' size mismatch at index 0. expected 769, actual 767
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:28:32.455926 47319186158848 torch/_dynamo/guards.py:2611] [9/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0419 14:28:32.455926 47319186158848 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:32.455926 47319186158848 torch/_dynamo/guards.py:2611] [9/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:32.471529 47319186158848 torch/_dynamo/guards.py:2611] [13/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_157 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/models.py:157
V0419 14:28:32.471529 47319186158848 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:32.471529 47319186158848 torch/_dynamo/guards.py:2611] [13/1] [__recompiles]     - tensor 'L['h']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:32.514461 47319186158848 torch/_dynamo/guards.py:2611] [4/3] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0419 14:28:32.514461 47319186158848 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:32.514461 47319186158848 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:28:32.514461 47319186158848 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 769                                         
V0419 14:28:32.514461 47319186158848 torch/_dynamo/guards.py:2611] [4/3] [__recompiles]     - L['num_dst'] == 769                                         
V0419 14:28:32.555179 47319186158848 torch/_dynamo/guards.py:2611] [8/3] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0419 14:28:32.555179 47319186158848 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:32.555179 47319186158848 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:28:32.555179 47319186158848 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 769                                       
V0419 14:28:32.555179 47319186158848 torch/_dynamo/guards.py:2611] [8/3] [__recompiles]     - L['___stack2'] == 769                                       
V0419 14:28:32.569364 47319186158848 torch/_dynamo/guards.py:2611] [15/1] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:28:32.569364 47319186158848 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:32.569364 47319186158848 torch/_dynamo/guards.py:2611] [15/1] [__recompiles]     - L['data'][0] == 769                                         
V0419 14:28:35.795279 47319186158848 torch/_dynamo/guards.py:2611] [29/1] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_572 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:572
V0419 14:28:35.795279 47319186158848 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:35.795279 47319186158848 torch/_dynamo/guards.py:2611] [29/1] [__recompiles]     - L['___stack0'] == 769                                       
V0419 14:28:35.812860 47319186158848 torch/_dynamo/guards.py:2611] [30/2] [__recompiles] Recompiling function torch_dynamo_resume_in_add_edges_at_573 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:573
V0419 14:28:35.812860 47319186158848 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:35.812860 47319186158848 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - L['___stack0'] == 769                                       
V0419 14:28:35.812860 47319186158848 torch/_dynamo/guards.py:2611] [30/2] [__recompiles]     - tensor 'L['u']' size mismatch at index 0. expected 1566, actual 1546
V0419 14:28:36.071899 47319186158848 torch/_dynamo/guards.py:2611] [34/3] [__recompiles] Recompiling function cat in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:222
V0419 14:28:36.071899 47319186158848 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:36.071899 47319186158848 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - 2 <= L['seq'][0].size()[0] <= 2147483645                      # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:36.071899 47319186158848 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 1566, actual 1546
V0419 14:28:36.071899 47319186158848 torch/_dynamo/guards.py:2611] [34/3] [__recompiles]     - tensor 'L['seq'][1]' size mismatch at index 0. expected 1566, actual 1546
V0419 14:28:36.306932 47319186158848 torch/_dynamo/guards.py:2611] [4/4] [__recompiles] Recompiling function create_unitgraph_from_coo in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1254
V0419 14:28:36.306932 47319186158848 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:36.306932 47319186158848 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 1546
V0419 14:28:36.306932 47319186158848 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:28:36.306932 47319186158848 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - L['num_dst'] == 769                                         
V0419 14:28:36.306932 47319186158848 torch/_dynamo/guards.py:2611] [4/4] [__recompiles]     - tensor 'L['row']' size mismatch at index 0. expected 0, actual 1546
V0419 14:28:36.345615 47319186158848 torch/_dynamo/guards.py:2611] [8/4] [__recompiles] Recompiling function torch_dynamo_resume_in_create_unitgraph_from_coo_at_1296 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:1296
V0419 14:28:36.345615 47319186158848 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:36.345615 47319186158848 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 1546
V0419 14:28:36.345615 47319186158848 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' dispatch key set mismatch. expected DispatchKeySet(CPU, BackendSelect, ADInplaceOrView, AutogradCPU), actual DispatchKeySet(CUDA, BackendSelect, ADInplaceOrView, AutogradCUDA)
V0419 14:28:36.345615 47319186158848 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - L['___stack2'] == 769                                       
V0419 14:28:36.345615 47319186158848 torch/_dynamo/guards.py:2611] [8/4] [__recompiles]     - tensor 'L['col']' size mismatch at index 0. expected 0, actual 1546
V0419 14:28:36.367510 47319186158848 torch/_dynamo/guards.py:2611] [41/1] [__recompiles] Recompiling function spmatrix in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/sparse/sparse_matrix.py:763
V0419 14:28:36.367510 47319186158848 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:36.367510 47319186158848 torch/_dynamo/guards.py:2611] [41/1] [__recompiles]     - tensor 'L['indices']' stride mismatch at index 0. expected 3986, actual 3946
V0419 14:28:36.643588 47319186158848 torch/_dynamo/guards.py:2611] [47/1] [__recompiles] Recompiling function arange in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:406
V0419 14:28:36.643588 47319186158848 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:36.643588 47319186158848 torch/_dynamo/guards.py:2611] [47/1] [__recompiles]     - L['stop'] == 769                                            
V0419 14:28:36.867354 47319186158848 torch/_dynamo/guards.py:2611] [48/1] [__recompiles] Recompiling function in_degrees in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:702
V0419 14:28:36.867354 47319186158848 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:36.867354 47319186158848 torch/_dynamo/guards.py:2611] [48/1] [__recompiles]     - tensor 'L['v']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:36.885876 47319186158848 torch/_dynamo/guards.py:2611] [50/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2878
V0419 14:28:36.885876 47319186158848 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:36.885876 47319186158848 torch/_dynamo/guards.py:2611] [50/1] [__recompiles]     - tensor 'L['vid']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:36.902423 47319186158848 torch/_dynamo/guards.py:2611] [51/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2925 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2925
V0419 14:28:36.902423 47319186158848 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:36.902423 47319186158848 torch/_dynamo/guards.py:2611] [51/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.142366 47319186158848 torch/_dynamo/guards.py:2611] [52/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2926 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2926
V0419 14:28:37.142366 47319186158848 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.142366 47319186158848 torch/_dynamo/guards.py:2611] [52/1] [__recompiles]     - tensor 'L['vid_tensor']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.165980 47319186158848 torch/_dynamo/guards.py:2611] [53/1] [__recompiles] Recompiling function has_nodes in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph_index.py:444
V0419 14:28:37.165980 47319186158848 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.165980 47319186158848 torch/_dynamo/guards.py:2611] [53/1] [__recompiles]     - tensor 'L['vids']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.185180 47319186158848 torch/_dynamo/guards.py:2611] [54/1] [__recompiles] Recompiling function torch_dynamo_resume_in_has_nodes_at_2930 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:2930
V0419 14:28:37.185180 47319186158848 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.185180 47319186158848 torch/_dynamo/guards.py:2611] [54/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.601741 47319186158848 torch/_dynamo/guards.py:2611] [55/1] [__recompiles] Recompiling function sum in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:152
V0419 14:28:37.601741 47319186158848 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.601741 47319186158848 torch/_dynamo/guards.py:2611] [55/1] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.684191 47319186158848 torch/_dynamo/guards.py:2611] [57/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0419 14:28:37.684191 47319186158848 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.684191 47319186158848 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=1
V0419 14:28:37.684191 47319186158848 torch/_dynamo/guards.py:2611] [57/2] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.702549 47319186158848 torch/_dynamo/guards.py:2611] [61/2] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0419 14:28:37.702549 47319186158848 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.702549 47319186158848 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0419 14:28:37.702549 47319186158848 torch/_dynamo/guards.py:2611] [61/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.721412 47319186158848 torch/_dynamo/guards.py:2611] [62/2] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0419 14:28:37.721412 47319186158848 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.721412 47319186158848 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0419 14:28:37.721412 47319186158848 torch/_dynamo/guards.py:2611] [62/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.740699 47319186158848 torch/_dynamo/guards.py:2611] [63/2] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0419 14:28:37.740699 47319186158848 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.740699 47319186158848 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=1
V0419 14:28:37.740699 47319186158848 torch/_dynamo/guards.py:2611] [63/2] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.758634 47319186158848 torch/_dynamo/guards.py:2611] [64/2] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0419 14:28:37.758634 47319186158848 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.758634 47319186158848 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=1
V0419 14:28:37.758634 47319186158848 torch/_dynamo/guards.py:2611] [64/2] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.777844 47319186158848 torch/_dynamo/guards.py:2611] [65/2] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0419 14:28:37.777844 47319186158848 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.777844 47319186158848 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=1
V0419 14:28:37.777844 47319186158848 torch/_dynamo/guards.py:2611] [65/2] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.821835 47319186158848 torch/_dynamo/guards.py:2611] [68/2] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0419 14:28:37.821835 47319186158848 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.821835 47319186158848 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=1
V0419 14:28:37.821835 47319186158848 torch/_dynamo/guards.py:2611] [68/2] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.835521 47319186158848 torch/_dynamo/guards.py:2611] [70/2] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0419 14:28:37.835521 47319186158848 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.835521 47319186158848 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=1
V0419 14:28:37.835521 47319186158848 torch/_dynamo/guards.py:2611] [70/2] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.853381 47319186158848 torch/_dynamo/guards.py:2611] [74/2] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0419 14:28:37.853381 47319186158848 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.853381 47319186158848 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=1
V0419 14:28:37.853381 47319186158848 torch/_dynamo/guards.py:2611] [74/2] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:37.869843 47319186158848 torch/_dynamo/guards.py:2611] [77/2] [__recompiles] Recompiling function zeros in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:287
V0419 14:28:37.869843 47319186158848 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:37.869843 47319186158848 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 769                                        
V0419 14:28:37.869843 47319186158848 torch/_dynamo/guards.py:2611] [77/2] [__recompiles]     - L['shape'][0] == 769                                        
V0419 14:28:38.046498 47319186158848 torch/_dynamo/guards.py:2611] [79/2] [__recompiles] Recompiling function zerocopy_to_dgl_ndarray_for_write in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:454
V0419 14:28:38.046498 47319186158848 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:38.046498 47319186158848 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:38.046498 47319186158848 torch/_dynamo/guards.py:2611] [79/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:38.089057 47319186158848 torch/_dynamo/guards.py:2611] [80/2] [__recompiles] Recompiling function check_is_view in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:441
V0419 14:28:38.089057 47319186158848 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:38.089057 47319186158848 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:38.089057 47319186158848 torch/_dynamo/guards.py:2611] [80/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:38.101202 47319186158848 torch/_dynamo/guards.py:2611] [81/2] [__recompiles] Recompiling function torch_dynamo_resume_in_check_is_view_at_443 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:443
V0419 14:28:38.101202 47319186158848 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:38.101202 47319186158848 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:38.101202 47319186158848 torch/_dynamo/guards.py:2611] [81/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:38.114459 47319186158848 torch/_dynamo/guards.py:2611] [82/2] [__recompiles] Recompiling function torch_dynamo_resume_in_zerocopy_to_dgl_ndarray_for_write_at_461 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/tensor.py:461
V0419 14:28:38.114459 47319186158848 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:38.114459 47319186158848 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:38.114459 47319186158848 torch/_dynamo/guards.py:2611] [82/2] [__recompiles]     - tensor 'L['input']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:38.132307 47319186158848 torch/_dynamo/guards.py:2611] [88/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:569
V0419 14:28:38.132307 47319186158848 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:38.132307 47319186158848 torch/_dynamo/guards.py:2611] [88/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:38.228867 47319186158848 torch/_dynamo/guards.py:2611] [89/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_590 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:590
V0419 14:28:38.228867 47319186158848 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:38.228867 47319186158848 torch/_dynamo/guards.py:2611] [89/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:38.305630 47319186158848 torch/_dynamo/guards.py:2611] [90/1] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:418
V0419 14:28:38.305630 47319186158848 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:38.305630 47319186158848 torch/_dynamo/guards.py:2611] [90/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:39.057880 47319186158848 torch/_dynamo/guards.py:2611] [93/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_432 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:432
V0419 14:28:39.057880 47319186158848 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:39.057880 47319186158848 torch/_dynamo/guards.py:2611] [93/1] [__recompiles]     - L['shape'][1] == 769                                        
V0419 14:28:40.460018 47319186158848 torch/_dynamo/guards.py:2611] [94/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_595 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:595
V0419 14:28:40.460018 47319186158848 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:40.460018 47319186158848 torch/_dynamo/guards.py:2611] [94/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:40.730375 47319186158848 torch/_dynamo/guards.py:2611] [95/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_604 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:604
V0419 14:28:40.730375 47319186158848 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:40.730375 47319186158848 torch/_dynamo/guards.py:2611] [95/1] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:41.493395 47319186158848 torch/_dynamo/guards.py:2611] [96/1] [__recompiles] Recompiling function torch_dynamo_resume_in_forward_at_648 in /work2/09808/mp46753/frontera/ML_Project/VQGraph/vq.py:648
V0419 14:28:41.493395 47319186158848 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:41.493395 47319186158848 torch/_dynamo/guards.py:2611] [96/1] [__recompiles]     - tensor 'L['___stack0']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:41.606913 47319186158848 torch/_dynamo/guards.py:2611] [57/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/view.py:82
V0419 14:28:41.606913 47319186158848 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:41.606913 47319186158848 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.606913 47319186158848 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:41.606913 47319186158848 torch/_dynamo/guards.py:2611] [57/3] [__recompiles]     - tensor 'L['val']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.627840 47319186158848 torch/_dynamo/guards.py:2611] [61/3] [__recompiles] Recompiling function __setitem__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:690
V0419 14:28:41.627840 47319186158848 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:41.627840 47319186158848 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.627840 47319186158848 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:41.627840 47319186158848 torch/_dynamo/guards.py:2611] [61/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.649111 47319186158848 torch/_dynamo/guards.py:2611] [62/3] [__recompiles] Recompiling function update_column in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:772
V0419 14:28:41.649111 47319186158848 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:41.649111 47319186158848 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.649111 47319186158848 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:41.649111 47319186158848 torch/_dynamo/guards.py:2611] [62/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.670683 47319186158848 torch/_dynamo/guards.py:2611] [63/3] [__recompiles] Recompiling function create in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:472
V0419 14:28:41.670683 47319186158848 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:41.670683 47319186158848 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.670683 47319186158848 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:41.670683 47319186158848 torch/_dynamo/guards.py:2611] [63/3] [__recompiles]     - tensor 'L['data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.690087 47319186158848 torch/_dynamo/guards.py:2611] [64/3] [__recompiles] Recompiling function __init__ in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:210
V0419 14:28:41.690087 47319186158848 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:41.690087 47319186158848 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.690087 47319186158848 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:41.690087 47319186158848 torch/_dynamo/guards.py:2611] [64/3] [__recompiles]     - tensor 'L['storage']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.711548 47319186158848 torch/_dynamo/guards.py:2611] [65/3] [__recompiles] Recompiling function infer_scheme in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/frame.py:151
V0419 14:28:41.711548 47319186158848 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:41.711548 47319186158848 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.711548 47319186158848 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:41.711548 47319186158848 torch/_dynamo/guards.py:2611] [65/3] [__recompiles]     - tensor 'L['tensor']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.760877 47319186158848 torch/_dynamo/guards.py:2611] [68/3] [__recompiles] Recompiling function func in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:213
V0419 14:28:41.760877 47319186158848 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:41.760877 47319186158848 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.760877 47319186158848 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:41.760877 47319186158848 torch/_dynamo/guards.py:2611] [68/3] [__recompiles]     - tensor 'L['x']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.776795 47319186158848 torch/_dynamo/guards.py:2611] [70/3] [__recompiles] Recompiling function torch_dynamo_resume_in_gspmm_at_75 in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/ops/spmm.py:75
V0419 14:28:41.776795 47319186158848 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:41.776795 47319186158848 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.776795 47319186158848 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:41.776795 47319186158848 torch/_dynamo/guards.py:2611] [70/3] [__recompiles]     - tensor 'L['lhs_data']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.795770 47319186158848 torch/_dynamo/guards.py:2611] [74/3] [__recompiles] Recompiling function forward in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/backend/pytorch/sparse.py:163
V0419 14:28:41.795770 47319186158848 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:41.795770 47319186158848 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
V0419 14:28:41.795770 47319186158848 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' size mismatch at index 0. expected 769, actual 767
V0419 14:28:41.795770 47319186158848 torch/_dynamo/guards.py:2611] [74/3] [__recompiles]     - tensor 'L['X']' requires_grad mismatch. expected requires_grad=0
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:28:41.974969 47319186158848 torch/_dynamo/guards.py:2611] [15/2] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:28:41.974969 47319186158848 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:41.974969 47319186158848 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 767                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:41.974969 47319186158848 torch/_dynamo/guards.py:2611] [15/2] [__recompiles]     - L['data'][0] == 769                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:28:45.370175 47319186158848 torch/_dynamo/guards.py:2611] [15/3] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:28:45.370175 47319186158848 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:45.370175 47319186158848 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 776                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:45.370175 47319186158848 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 767                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:45.370175 47319186158848 torch/_dynamo/guards.py:2611] [15/3] [__recompiles]     - L['data'][0] == 769                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:28:45.613180 47319186158848 torch/_dynamo/guards.py:2611] [15/4] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:28:45.613180 47319186158848 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:45.613180 47319186158848 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 777                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:45.613180 47319186158848 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 776                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:45.613180 47319186158848 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 767                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:45.613180 47319186158848 torch/_dynamo/guards.py:2611] [15/4] [__recompiles]     - L['data'][0] == 769                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:28:48.995590 47319186158848 torch/_dynamo/guards.py:2611] [15/5] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:28:48.995590 47319186158848 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:48.995590 47319186158848 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 784                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:48.995590 47319186158848 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 777                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:48.995590 47319186158848 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 776                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:48.995590 47319186158848 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 767                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:48.995590 47319186158848 torch/_dynamo/guards.py:2611] [15/5] [__recompiles]     - L['data'][0] == 769                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:28:52.407963 47319186158848 torch/_dynamo/guards.py:2611] [15/6] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:28:52.407963 47319186158848 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:52.407963 47319186158848 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 779                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:52.407963 47319186158848 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 784                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:52.407963 47319186158848 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 777                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:52.407963 47319186158848 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 776                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:52.407963 47319186158848 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 767                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:52.407963 47319186158848 torch/_dynamo/guards.py:2611] [15/6] [__recompiles]     - L['data'][0] == 769                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:28:56.375554 47319186158848 torch/_dynamo/guards.py:2611] [15/7] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:28:56.375554 47319186158848 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:56.375554 47319186158848 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 776                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:56.375554 47319186158848 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 773                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:56.375554 47319186158848 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 779                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:56.375554 47319186158848 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 784                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:56.375554 47319186158848 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 777                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:56.375554 47319186158848 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 767                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:56.375554 47319186158848 torch/_dynamo/guards.py:2611] [15/7] [__recompiles]     - L['data'][0] == 769                                         
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
V0419 14:28:59.788933 47319186158848 torch/_dynamo/guards.py:2611] [15/8] [__recompiles] Recompiling function _dispatch in /work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65
V0419 14:28:59.788933 47319186158848 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     triggered by the following guard failure(s):
V0419 14:28:59.788933 47319186158848 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 759                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:59.788933 47319186158848 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 776                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:59.788933 47319186158848 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 773                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:59.788933 47319186158848 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 779                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:59.788933 47319186158848 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 784                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:59.788933 47319186158848 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 777                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:59.788933 47319186158848 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 767                                           # _dynamo/output_graph.py:452 in init_ambient_guards
V0419 14:28:59.788933 47319186158848 torch/_dynamo/guards.py:2611] [15/8] [__recompiles]     - L['data'][0] == 769                                         
W0419 14:28:59.789148 47319186158848 torch/_dynamo/convert_frame.py:762] [15/8] torch._dynamo hit config.cache_size_limit (8)
W0419 14:28:59.789148 47319186158848 torch/_dynamo/convert_frame.py:762] [15/8]    function: '_dispatch' (/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/utils/internal.py:65)
W0419 14:28:59.789148 47319186158848 torch/_dynamo/convert_frame.py:762] [15/8]    last reason: L['data'][0] == 769                                         
W0419 14:28:59.789148 47319186158848 torch/_dynamo/convert_frame.py:762] [15/8] To log all recompilation reasons, use TORCH_LOGS="recompiles".
W0419 14:28:59.789148 47319186158848 torch/_dynamo/convert_frame.py:762] [15/8] To diagnose recompilation issues, see https://pytorch.org/docs/main/torch.compiler_troubleshooting.html.
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/dataloading/dataloader.py:1144: DGLWarning: Dataloader CPU affinity opt is not enabled, consider switching it on (see enable_cpu_affinity() or CPU best practices for DGL [https://docs.dgl.ai/tutorials/cpu/cpu_best_practises.html])
  dgl_warning(
/work2/09808/mp46753/frontera/ML_Project/miniconda3/envs/txgnn_env/lib/python3.8/site-packages/dgl/heterograph.py:92: DGLWarning: Recommend creating graphs by `dgl.graph(data)` instead of `dgl.DGLGraph(data)`.
  dgl_warning(
Compiling model with torch.compile()...
Saved sparsity snapshot to outputs_compiled/tran/citeseer/SAGE/seed_2/sparsity_before.csv
                               layer_name  num_elements    sparsity
0  _orig_mod.encoder.graph_layer_1.weight      13712209   49.998815
1    _orig_mod.encoder.graph_layer_1.bias          3703  100.000000
2  _orig_mod.encoder.graph_layer_2.weight        473984   50.087556
3    _orig_mod.encoder.graph_layer_2.bias           128  100.000000
4      _orig_mod.encoder.decoder_1.weight      13712209   49.989539
5        _orig_mod.encoder.decoder_1.bias          3703   50.391574
6      _orig_mod.encoder.decoder_2.weight      13712209   50.005422
7        _orig_mod.encoder.decoder_2.bias          3703   49.635431
8         _orig_mod.encoder.linear.weight           768   48.307292
9           _orig_mod.encoder.linear.bias             6   50.000000
Saved sparsity snapshot to outputs_compiled/tran/citeseer/SAGE/seed_2/sparsity_after.csv
                               layer_name  num_elements   sparsity
0  _orig_mod.encoder.graph_layer_1.weight      13712209  54.093268
1    _orig_mod.encoder.graph_layer_1.bias          3703  80.988388
2  _orig_mod.encoder.graph_layer_2.weight        473984  49.799782
3    _orig_mod.encoder.graph_layer_2.bias           128  60.156250
4      _orig_mod.encoder.decoder_1.weight      13712209  49.986629
5        _orig_mod.encoder.decoder_1.bias          3703  50.094518
6      _orig_mod.encoder.decoder_2.weight      13712209  47.490284
7        _orig_mod.encoder.decoder_2.bias          3703  24.979746
8         _orig_mod.encoder.linear.weight           768  48.177083
9           _orig_mod.encoder.linear.bias             6  33.333333
Saved forward timing report to outputs_compiled/tran/citeseer/SAGE/seed_2/forward_times.csv
                         layer_name  avg_forward_time_ms  num_calls
0         _orig_mod.encoder.dropout             0.002197       3005
1   _orig_mod.encoder.graph_layer_1             0.002903       3005
2   _orig_mod.encoder.graph_layer_2             0.002408       3005
3       _orig_mod.encoder.decoder_1             0.003056       3005
4       _orig_mod.encoder.decoder_2             0.002350       3005
5          _orig_mod.encoder.linear             0.000553       3005
6   _orig_mod.encoder.vq.project_in             0.002214       3005
7  _orig_mod.encoder.vq.project_out             0.003017       3005
8    _orig_mod.encoder.vq._codebook             0.002970       3005
CPU Memory Usage: 1258.09 MB
Max GPU Memory Used: 1362.30 MB
